<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bill</class>
 <widget class="QWidget" name="Bill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diner Tablet</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QFrame {
    border-radius: 40px;
    background: rgba(255,255,255,40%);
}

QPushButton {
    min-height: 2.0em;
    font: 2em;
    margin: 0 1px 0 1px;
    color: white;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #581702, stop: 1 #7f3016); 
    border-style: outset;
    border-radius: 10px;
    border-width: 1px;
    border-color: #3e170b;
}
 
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #aa401e stop: 1 #581702);
}

QTabWidget,QTabWidget::pane,QTabBar { 
    background: transparent; border: 0px; 
}

QTabBar::tab {
    background-color: rgb(0, 0, 0, 0);
    padding: 2px;
    border: 2px;
    text-align: center;
    height: 64px;
    width: 64px;
    border-style: outset;
    border-radius: 10px;
    border-width: 1px;
}

QTabWidget#tabWidget_sidebar {
    background-color: rgba(255, 255, 255, 0);
}

QTabWidget#tabWidget_sidebar::tab-bar {
   alignment: center;
}
 
QTabWidget#tabWidget_sidebar::pane {
   background-color: rgb(255, 255, 255, 0);
}

QTabBar::tab:selected {
    background-color: rgba(255, 255, 255, 100);
}
 
QTabBar::tab:hover {
    background-color: rgba(255, 255, 255, 150);
}
 
QTextBrowser {
    background-color : transparent;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="labelBill">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">#label {
}</string>
     </property>
     <property name="text">
      <string>Order</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Order 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="textBrowser">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line">
     <property name="styleSheet">
      <string notr="true">#line {
 border: 1px solid black;
 border-radius: 10px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #581702, stop: 1 #7f3016); 
}</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    font: bold 16px;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacerEnd">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>365</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonRequest">
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    font: bold 16px;
}</string>
       </property>
       <property name="text">
        <string>Request</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
